tests/auto/shared/perfparsertestclient.cpp

```diff
@@ -104,7 +104,7 @@ void PerfParserTestClient::extractTrace(QIODevice *device)
         case LocationDefinition: {
             qint32 id;
             LocationEvent location;
-            stream >> id >> location.address >> location.file >> location.pid >> location.line
+            stream >> id >> location.address >> location.relAddr >> location.file >> location.pid >> location.line
```

Comment:
now also print these values to the text form and include the new values in the expected files - just run the test and copy the "actual" file over to the "expected" and double check that the values are sane (e.g. manually via objdump)